### PR TITLE
[core/ui] add more padding into auction card

### PR DESCRIPTION
**Pull request by vuvu2259**

### CODE REVIEW

File: core/ui/src/components/Modal/index.less [hunk 1]
```diff
@@ -15,7 +15,8 @@
     left: 50%;
     transform: translate(-50%, -50%);
 
-    padding: 20px;
+    padding: 40px;
+    padding-bottom: 20px;
```
Comment (haVincy) [inline core/ui/src/components/Modal/index.less:19]: Why don't we just
```less
padding: 40px 40px 20px 40px;
```

Comment (vuvu2259) [reply core/ui/src/components/Modal/index.less:19]: It's not friendly for me :D

Comment (vuvu2259) [reply core/ui/src/components/Modal/index.less:19]: I will update for u ^^

Comment (vuvu2259) [reply core/ui/src/components/Modal/index.less]: updated

Comment (haVincy) [reply core/ui/src/components/Modal/index.less]: If you prefer specific declaration, then shouldn't mix up with `padding` but 
```less
padding-top: 40px;
padding-bottom: 20px;
padding-left: 40px;
padding-right: 40px;
```

**Review by haVincy — APPROVED** [review]

**Pull request merged**